docs/admin/accessing-the-api.md

```diff
@@ -53,7 +53,9 @@ If the request cannot be authenticated, it is rejected with HTTP status code 401
 Otherwise, the user is authenticated as a specific `username`, and the user name
 is available to subsequent steps to use in their decisions.  Some authenticators
 may also provide the group memberships of the user, while other authenticators
-do not (and expect the authorizer to determine these).
+do not; an authorizer may determine as well as use groups (or any other
+auxiliary concepts), as a private detail of that authorizer (none of the standard
+authorizers determines groups).
```

Comment:
I agree that sentence is a long and that could use fixing.  I disagree with saying that the authenticators that do not provide groups "expect" the authorizer to do so.  Frankly, the authenticators do not give a damn.  And none of the supplied authorizers determine groups.  So saying it is "expected" seems to me like it is not a good characterization of the situation.

It seems to me that if the operator selects an authenticator that does not supply groups then that operator had better use authorizers that either do not need groups or determine groups themselves.  In the latter case the groups are a private detail of the authorizer, which can theoretically have all sorts of private details.